### PR TITLE
RET 2074 - ET1/ET3 Extension- After ET1 has been processed and accepted

### PR DESCRIPTION
**Pull request by brendanmchugh**
### JIRA link

https://tools.hmcts.net/jira/browse/RET-2074

### Change description

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

File: src/main/controllers/CaseDocumentController.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const { Logger } = require('@hmcts/nodejs-logging');
+
+const logger = Logger.getLogger('CaseDocumentController');
+
+export default class CaseDocumentController {
+  public async get(req: AppRequest, res: Response): Promise<void> {
+    try {
+      if (!req.params?.docId) {
+        logger.info('bad request parameter');
+        return res.redirect('/citizen-hub/' + req.session?.userCase?.id);
```
Comment (Itay-Kirsh) [inline src/main/controllers/CaseDocumentController.ts:15]: Maybe lead them to the not-found page here too, you don't want to hide an error from the user, even if it isn't an exception.

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

**Review by robin-ieong — APPROVED** [review]

File: src/main/controllers/CaseDocumentController.ts (hunk 1 above)
Comment (brendanmchugh) [reply src/main/controllers/CaseDocumentController.ts]: updated

File: src/main/resources/locales/en/translation/citizen-hub.json [hunk 2]
```diff
@@ -6,11 +6,9 @@
   "received": "Response received",
   "details": "Your hearing details",
   "decision": "Your claim decision",
-
```
[the remark this answers is not in the document]
Comment (brendanmchugh) [reply src/main/resources/locales/en/translation/citizen-hub.json:9]: Its the default file formatting for json files with vscode.

**Review by Itay-Kirsh — APPROVED** [review]

**Comment by brendanmchugh** [comment]
Superseded by pull request 352

**Pull request closed without merging**